lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -11,6 +12,7 @@
  */
 
 use Lizmap\Project;
```

Comment:
```suggestion
use Lizmap\Project\Project;
```
To avoid `Project\Project`